### DISCUSSION TITLE
Cannot update email because profile update no longer lists United States

### DISCUSSION THREAD

**Discussion by astrodad** [ovh/manager#9654] [General]
Hi,

I'm trying to update my profile, but there is no dropdown entry for US, which means the state selection doesn't work, which means I get an error trying to update anything. I have an open ticket that's been siting for days. Who needs to look at this?

![Screenshot 2023-07-14 at 15 48 37](https://github.com/ovh/manager/assets/1663239/68788859-7963-4b5f-8cda-040dda6fe1ff)

**Comment by yan-hic** [comment]
Same here + there is no `+1` for phone country dropdown. While I can pick Canada (+1 too), the form doesn't validate the number and changes can't be saved.
@astrodad since you had a ticket and it's not resolved yet, I guess we are the only 2 US customers and OVH doesn't care....

**Reply by astrodad** [reply]
I ended up getting an email and they resolved it. Support took FOREVER to get back to me, but once they did it was resolved quickly.
